Makefile

```diff
@@ -0,0 +1,23 @@
+.DEFAULT_GOAL := help
+
+ifndef TAG
```

Comment:
Why not a simple `?=`?

Reply:
Because I don't know the syntax of Makefile good enough :D in the mean time, I find this form more comprehensible for newcomers!

Reply:
I find it much harder to read this way (very verbose) but eh, whatever. :-P